forc/src/ops/forc_clean.rs

```diff
@@ -31,12 +31,16 @@ pub fn clean(command: CleanCommand) -> Result<()> {
 
     // Run `cargo clean`, forwarding stdout and stderr (`cargo clean` doesn't appear to output
     // anything as of writing this).
-    process::Command::new("cargo")
-        .arg("clean")
-        .stderr(process::Stdio::inherit())
-        .stdout(process::Stdio::inherit())
-        .output()
-        .map_err(|e| e)?;
+    if find_cargo_manifest_dir(&this_dir).is_some() {
+        process::Command::new("cargo")
+            .arg("clean")
+            .stderr(process::Stdio::inherit())
+            .stdout(process::Stdio::inherit())
+            .output()
+            .map_err(|e| e)?;
+    } else {
+        println_yellow_err("WARNING: use of forc clean without a Cargo.toml file in ANY parent directory will result in a bypass of the cargo clean command.").unwrap();
```

Comment:
On a second look, I don't think we want this warning branch :smiling_face_with_tear: Some forc projects may have simply not run any `forc test`s yet and as a result they haven't generated a `Cargo.toml`. In those cases it's fine to just do the `out` directory cleanup and skip this step without warning.